apps/www/_blog/2023-08-11-supabase-soc2-hipaa.mdx

```diff
@@ -52,7 +52,6 @@ We receive many requests from users who want to build healthcare apps on top of
 <Img
   src="/images/blog/launch-week-8/day-5/soc2-to-hipaa.png"
   alt="soc2 to HIPAA"
-  className="!max-h-[450px]"
```

Comment:
Hey, I think it was typo. Instead of completely removing it, please add the class **max-h-[450px]**. Also, can you attach screenshots of before and after? Thanks!

Reply:
Hey @AmanMollah, I added screenshots of the before and after. 👍

> Instead of completely removing it, please add the class max-h-[450px]

The issue isn't actually fixed unless I remove that line.
And a tall image is preferable rather than a cropped one, in this case.